src/install/vm/deployment/new.mdx

```diff
@@ -0,0 +1,30 @@
+---
+componentType: appInfoConfigOption
+optionType: deployment 
+headingText: Upload new vulnerabilities through the Dependabot Webhook Integration
+---
+
+## Prerequisites
+
+- A New Relic [license key](/docs/accounts-partnerships/install-new-relic/account-setup/license-key) for the account you want to report data to.
+- Organization/Repository administration access to your GitHub account(s)
+    - Organization webhooks are strongly recommended. It’s much easier to administer a single webhook per org than to create separate rules for each repository.
+
+## Create the webhook
+
+To complete the webhook you need:
```

Comment:
prob more instructional/procedural sounding intro, like just 'To create the webhook:'